### ISSUE TITLE
ResultCode 151 unknown in the client. Please file a github issue

### ISSUE THREAD

**Issue by madejejej** [aerospike/aerospike-client-ruby#97]
We are getting this error for some of our requests. It says in [the error codes reference](https://www.aerospike.com/docs/dev_reference/error_codes.html) that this error means that the batch max requests has been exceeded. Could the client handle this by sending multiple requests in parallel, each with a proper batch size?

**Comment by khaf** [comment]
No not currently. We may consider it for a future feature, but it's not a priority at this very moment.
